uk.co.compucorp.civicrm.tasksassignments/CRM/Activity/Service/ActivityService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CRM_Activity_Service_ActivityService {
+
+  /**
+   * Returns the activity types for a certain component. Default limit is 25
+   *
+   * @param $componentName
+   *  The name of the component
+   * @param int $limit
+   *  The maximium number of results to return
+   *
+   * @return array
+   */
+  public static function findByComponent($componentName, $limit = 0) {
```

Comment:
Sorry, what I meant is that while the method's description says the default is 25, the default is unlimited, because it is set as 0.  So the method's description is not accurate, maybe better to delete "Default limit is 25", or update it to "Default limit is 0 (unlimited)".

Again, sorry for the mix-up!